modules/op-release-notes-1-15.adoc

```diff
@@ -0,0 +1,451 @@
+* Users can add messages while approving or rejecting the `approvalTask`.
+* You can add a timeout setting to the `approvalTask` specification. If the required number of approvals is not provided during this time period, the pipeline run fails.
+
+:FeatureName: The manual approval gate
+include::snippets/technology-preview.adoc[]
+
+
+[id="cli-new-features-1-15_{context}"]
+=== CLI
+
+* With this update, the `tkn` command line utility supports the `-E` or `--exit-with-pipelinerun-error` option for the `pipeline showlog` command. With this option, the command line utility returns an error code of `0` if the pipeline run completed successfully, `1` if the pipeline run ended with an error, and `2` if the status of the pipeline run is unknown.
+
+* With this update, the `tkn` command line utility supports the `--label` option for the `bundle push` command. With this option, you can provide the value of a label in the `<label-name>=<value>` format; the utility adds the label to the OCI image that it creates. You can use this option several times to provide several labels for the same image.
+
+[id="pac-new-features-1-15_{context}"]
+=== {pac}
+
+* With this update, when using {pac}, you can set a `pipelinesascode.tekton.dev/on-comment` annotation on a pipeline run to start the pipeline run when a developer adds a matching comment to a pull request. This setting is supported only for pull requests and only for GitHub and GitLab repository providers.
+
+:FeatureName: Matching a comment event to a pipeline run
+include::snippets/technology-preview.adoc[]
+
+* With this update, when using {pac}, you can enter the `/test <pipeline_run_name>` comment on a pull request to start any {pac} pipeline run on the repository, whether or not it was triggered by an event for this pipeline run. This feature is Technological preview only.
+
+* With this update, when providing a `/test` or `/retest` command for {pac} in a Git request comment, you can now set any standard or custom parameters for the pipeline run.
++
+.Example commands in a Git request comment
+----
+/test pipelinerun1 revision=main param1="value1" param2="value \"value2\" with quotes"
+----
++
+This command runs the `pipelinerun1` pipeline run on the `main` branch instead of the pull request branch.
++
+----
+/test checker target_branch=backport-branch
+----
++
+This command runs the `checker` pipeline run on a backport (cherry-pick) of the pull request to the `backport-branch` branch.
+
+* With this update, when using {pac}, you can create a global `Repository` CR with the name `pipelines-as-code` in the namespace in which {pipelines-shortname} is installed, normally `openshift-pipelines`. In this CR, you can set the configuration options that apply to all `Repository` CRs. You can override any of these default options by setting different values in the `Repository` CR for a particular repository.
++
+:FeatureName: The global `Repository` CR
+include::snippets/technology-preview.adoc[]
```

Comment:
`{FeatureName}` is coming through verbatim here in the preview. You can fix this by either wrapping in open block (if you want to keep it attached to the bullet's indentation):
```suggestion
+
--
:FeatureName: The global `Repository` CR
include::snippets/technology-preview.adoc[]
--
```
or detach it:
```suggestion

:FeatureName: The global `Repository` CR
include::snippets/technology-preview.adoc[]